packages/backend/src/createRedirect.ts

```diff
@@ -12,6 +18,10 @@ const buildUrl = (_baseUrl: string | URL, _targetUrl: string | URL, _returnBackU
   if (returnBackUrl) {
     res.searchParams.set('redirect_url', returnBackUrl.toString());
   }
+  // For cross-origin redirects, we need to pass the dev browser token for URL session syncing
+  if (_devBrowserToken && baseUrl.hostname !== res.hostname) {
+    res.searchParams.set(constants.Cookies.DevBrowser, _devBrowserToken);
```

Comment:
@BRKalow i think we should use the QueryParameters constants when adding search params instead of Cookies contants even though they both have the same value.
PR to resolve this: https://github.com/clerk/javascript/pull/3355